plugins/carbon_black_defense/plugin.spec.yaml

```diff
@@ -523,209 +136,398 @@ types:
-      type: netFlow
-    shortDescription:
-      title: Short Description
-      type: string
-    targetApp:
-      title: Target Hash
-      type: hash
+      description: "Netconn local port"
+      required: false
+    device_os_version:
+      title: "Device OS Version"
+      type: string
+      description: "Device OS version"
+      required: false
+    org_id:
+      title: "Org ID"
+      type: string
+      description: "Org ID"
+      required: false
+    device_name:
+      title: "Device Name"
+      type: "[]string"
+      description: "Device name"
+      required: false
+    backend_timestamp:
+      title: "Backend Timestamp"
+      type: date
+      description: "Backend timestamp"
+      required: false
+    parent_pid:
+      title: "Parent PID"
+      type: integer
+      description: "Parent PID"
+      required: false
+    device_policy:
+      title: "Device Policy"
+      type: string
+      description: "Device policy"
+      required: false
+    parent_reputation:
+      title: "Parent Reputation"
+      type: string
+      description: "Parent reputation"
+      required: false
 connection:
   url:
     title: URL
     type: string
     description: API URL
-    default: https://api.confer.net
+    default: https://defense.conferdeploy.net
     required: true
+    example: https://example.com
   api_key:
     title: API Key
     type: credential_secret_key
     description: API key
     required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  org_key:
+    title: Organization Key
+    type: string
+    description: Organization key
+    required: true
+    example: 1ABCDE5GH
   connector:
     title: Connector
     type: string
     description: Connector ID
     required: true
+    example: 1Y72FRRWVP
 actions:
   find_event:
     title: Find Event
     description: Retrieves all events matching the input search criteria. Response
       is a list of events in JSON format. Resulting events are sorted in descending
       order of time
     input:
-      hostName:
-        title: Hostname
-        description: Filter on hostnames case insensitive. For example hostName=win-IA9NQ1GN8OI
-          will match the hostname WIN-IA9NQ1GN8OI
-        type: string
-        required: false
-      hostNameExact:
-        title: Hostname Exact
-        description: Filter on the exact hostname. For example hostName=WIN-IA9NQ1GN8OI
-          will only return devices with the exact hostname WIN-IA9NQ1GN8OI but not
-          a host named win-IA9NQ1GN8OI
-        type: string
-        required: false
-      ownerName:
-        title: Owners Name
-        description: Filter on owner name case insensitive
-        type: string
-        required: false
-      ownerNameExact:
-        title: Owners Name Exact
-        description: Same as 'Owners Name' but with case sensitivity
-        type: string
+      device_name:
+        title: "Device Name"
+        description: "Device name"
+        type: "[]string"
```

Comment:
Why do you need array of string. Does the input accept list of devices?

Reply:
The API accept multiple values as an array for all input options. The plugin action will send all of these in a query.